app/cluster/integration_test.go

```diff
@@ -66,7 +66,7 @@ func run(m *testing.M) int {
 
 	database := "jobsdb"
 	// pulls an image, creates a container based on it and runs it
-	resourcePostgres, err := pool.Run("postgres", "11-alpine", []string{
+	resourcePostgres, err := pool.Run("postgres", "15-alpine", []string{
```

Comment:
Since we are still using postgres-11 in various environments, shouldn't we leave it as is until we upgrade them first?